### DISCUSSION TITLE
Ask about Req/Res History

### DISCUSSION THREAD

**Discussion by afrianjunior** [usebruno/bruno#1873] [General]
Hi @helloanoop I need to know what do you expect about request/response history.

Discussion will be scoped in:
- Storage model
- UI

I want to work on it.

**Comment by janos-r** [comment]
I agree, history is quite essential. Probably the main reason why you would use a GUI for requests anyway.

**Comment by sanjai0py** [comment]
Hey @afrianjunior and @janos-r, we have this feature in the Golden edition of Bruno. 

Here's the demo: 

https://github.com/user-attachments/assets/473afc98-78a3-41da-b78c-376a59234992

**Comment by janos-r** [comment]
Isn't " Request Response History" on the official roadmap for "version 2"? I would really hope it get's there for all users...
Btw I would expect it as a per-request slot history... Simply just to see example of responses after coming back to it on a different day.
